spot-ml/src/main/scala/org/apache/spot/dns/model/DNSSuspiciousConnectsModel.scala

```diff
@@ -205,15 +201,16 @@ object DNSSuspiciousConnectsModel {
                        userDomain: String,
                        url: String): TempFields = {
 
-    val DomainInfo(_, topDomainClass, subdomain, subdomainLength, subdomainEntropy, numPeriods) =
+    val DomainInfo(_, topDomainClass, _, subDomainLength, subDomainEntropy, numPeriods) =
```

Comment:
Same thing seems to be for createTempFields in DNSSuspiciousConnectsModel.scala, seems like it's only being used in unit tests.